### PR TITLE
OAK-1 update

### PR DESCRIPTION
**Pull request by cafemoloko**
- fixed link -> to OAK-1 page
- added dimensions and weight

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Looks great to me; thanks!

File: docs/source/index.rst [hunk 1]
```diff
@@ -178,7 +178,7 @@ Miscellaneous Designs
    :caption: USB Designs
 
    pages/BW1098OAK.rst
-   pages/BK1096.rst
```
Comment (VanDavv) [inline docs/source/index.rst:181]: Could you remove `BK1096.rst` page? It seems unused

**Review by VanDavv — CHANGES_REQUESTED** [review]

File: docs/source/index.rst (hunk 1 above)
Comment (cafemoloko) [reply docs/source/index.rst:181]: @VanDavv, thanks for noticing! I will use it again.

**Comment by Luxonis-David** [comment]
Can you also add link to the mechanical 3D model on the bottom of the page like we have for other prodcuts: 
https://github.com/luxonis/depthai-hardware/tree/master/Mechanical%20designs

**Comment by cafemoloko** [comment]
> Can you also add link to the mechanical 3D model on the bottom of the page like we have for other prodcuts:
> https://github.com/luxonis/depthai-hardware/tree/master/Mechanical%20designs

@Luxonis-David Thanks for the comment, link added

**Review by Luxonis-David — APPROVED** [review]

**Review by VanDavv — APPROVED** [review]

**Pull request merged**